client/state/reader/start/actions.js

```diff
@@ -123,3 +127,34 @@ export function requestRecommendations( originSiteId, originPostId, limit ) {
 		);
 	};
 }
+
+/**
+ * Triggers a network request to graduate the logged in user
+ * from the Reader following recommendations page.
+ *
+ * @return {Function} Action thunk
+ */
+export function requestGraduate() {
+	return ( dispatch ) => {
+		dispatch( {
+			type: READER_START_GRADUATE_REQUEST,
+		} );
+
+		debug( 'Graduating user from cold start' );
+
+		return wpcom.undocumented().graduateNewReader()
+			.then( ( data ) => {
+				dispatch( {
```

Comment:
So to confirm, are `READER_START_GRADUATE_REQUEST_SUCCESS` and `READER_START_GRADUATE_REQUEST_FAILURE` still necessary or no?